### ISSUE TITLE
not able to install

### ISSUE THREAD

**Issue by Anoiki** [krljg/lsystem#10]
I can't install the addon, I tried manually into the directory but nothing. I'm using  blender 3.4.1 on a Mac

**Comment by krljg** [comment]
Hi,
Have you copied the lsystem directory to the blender addons? It should look something like this:
![lsystem_directory](https://user-images.githubusercontent.com/14252670/213289626-fa420a75-084e-4b04-adfb-eb5673549e1e.png)

And then you need to enable the add-on in blender:
![blender_preferences_lsystem](https://user-images.githubusercontent.com/14252670/213289885-a2cd8528-74bc-4e0c-9ec3-95fe6b4a2fcb.png)

**Comment by Anoiki** [comment]
yes! I did. I have a Mac tho so I'm not using the steam app. But I pasted
it into the Blender Scripts addon folder and nothing.
I wonder if it has anything to do w trying to use it on a OS system?

Alejandro Figueredo Díaz-Perera
<https://www.engage-projects.com/alejandro-figueredo-diacuteaz-perera-bio.html>

34000 Pillows featured in Hyperallergic,
<https://hyperallergic.com/495661/34000-pillows-project-diaz-lewis/>
by Lorissa Reinhart
<https://hyperallergic.com/495661/34000-pillows-project-diaz-lewis/>

Landmarks Video screening at the University of Texas
<https://landmarks.utexas.edu/event/landmarks-video-presents-love-rebellious-bird-alejandro-figueredo-d%C3%ADaz-perera>
1-30 June 2019
<https://landmarks.utexas.edu/event/landmarks-video-presents-love-rebellious-bird-alejandro-figueredo-d%C3%ADaz-perera>

**Comment by krljg** [comment]
Could be I guess. I'm running Blender 3.4.1 on windows and that seems to work. Don't have a mac so I can't test it.

**Comment by covalspace** [comment]
Hi @Anoiki I'm on a mac and was able to get it working:

I downloaded the whole github repo to my downloads
Expanded the zip file (lsystem-master.zip)
Then I recompressed the lsystem subdirectory (to lsystem.zip)
Then I added the lsystem.zip file (compressed) to my addons from within the settings of blender.

Pleasant surprise is I am currently using Blender 4.2 and am able to run this add-on with no issues.

Great work @krljg !!!
